data/roles.js

```diff
@@ -629,157 +648,163 @@ const roleData = {
         "Diabolist": {
             alignment: "Mafia",
             description: [
-                "Can choose a victim and a target each night.",
-                "If the victim votes for the target, the victim will die."
+                "Chooses a player to be a victim and a target each night.",
+                "If the victim votes for the target in the village meeting the following day, the victim will die."
             ]
         },
         "Tailor": {
             alignment: "Mafia",
             description: [
-                "Gives out suits that make the wearer appear to be a different role."
+                "Gives out a suit each night that disguise the wearer's role identity.",
+                "Suits can be selected from any role within the current game."
             ],
         },
         "Actress": {
             alignment: "Mafia",
             description: [
-                "Learns a targets role and appears to be that role."
+                "Visits a player to appears as their role.",
+                "Learns chosen player's role."
             ],
         },
         "Prosecutor": {
             alignment: "Mafia",
             description: [
-                "Votes weight is 2."
+                "Vote weight is worth 2 players votes in village meeting."
             ],
         },
         "Fabricator": {
             alignment: "Mafia",
             description: [
-                "Once per night can give out an ill-fated version of many items."
+                "Gives out a fabricated item once per night.",
+                "Fabricated Guns and Knives will backfire against the player who used them.",
+                "Fabricated Armor, Crystals and Snowballs will be ineffective."
             ],
         },
         "Heartbreaker": {
             alignment: "Mafia",
             description: [
-                "Once per game can fall in love with another player.",
-                "If Heartbreaker dies, both players will die.",
+                "Fall in love with another player once per game.",
+                "Both players will die if Heartbreaker dies.",
             ],
         },
         "Yakuza": {
             alignment: "Mafia",
             description: [
-                "Can choose to sacrifice themself to convert one player to Mafioso.",
+                "Chooses to sacrifice self once per game to convert another player to Mafioso.",
             ],
         },
         "Necromancer": {
             alignment: "Mafia",
             description: [
-                "Once per game, visits one dead person during the night.",
-                "That person will be resurrected.",
+                "Visits a dead player during the night once per game.",
+                "That player will be resurrected the following day.",
+                "If players identity was revealed upon death, they will remain revealed when ressurected."
```

Comment:
Grammar. Should be "player's" instead of "players."
Also should probably be "role identity" for consistency.
Spelling. Should be "resurrected" instead of "ressurected."